### PR TITLE
add TryHub.de

### PR DESCRIPTION
**Pull request by MrManHD**
## Type of change

- [ x ] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [ x ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ x ] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [ x ] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/tryhub/manifest.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "social": {
+    "web": "https://tryhub.de",
+    "web_support": "https://tryhub.de/message-us",
+    "twitter": "TryHubDE",
+    "discord": "https://discord.gg/PFpuaMEK2E",
+    "teamspeak": "tryhub.de"
+  },
+  "discord": {
+    "server_id": 806169547408277564,
+    "rename_to_minecraft_name": true
```
Comment (JakobBruening) [inline minecraft_servers/tryhub/manifest.json:20]: Please change to `false`, as this is the default value.

**Review by JakobBruening — CHANGES_REQUESTED** [review]
Please compress your background image :)

File: minecraft_servers/tryhub/manifest.json (hunk 1 above)
Comment (MrManHD) [reply minecraft_servers/tryhub/manifest.json]: fixed

**Review by JakobBruening — APPROVED** [review]
Thanks! 👍🏼

**Pull request merged**